### PR TITLE
Add data access to project creation

### PR DESCRIPTION
**Pull request by mariarivera30**

### CODE REVIEW

File: pybossa/forms/dynamic_forms.py [hunk 1]
```diff
@@ -17,3 +20,27 @@ def build_field(field_type, description, validators, kwargs=None):
...
+    class ProjectFormExtraInputs(class_type):
+        pass
+
+    if data_access_levels:
+        data_access = Select2Field(
+            lazy_gettext('Access Level(s)'),
+            [validators.Required()],
+            choices=data_access_levels['valid_access_levels'],
+            default=[])
+        setattr(ProjectFormExtraInputs, 'data_access', data_access)
```
Comment (fiorda) [inline pybossa/forms/dynamic_forms.py]: You can do `ProjectFormExtraInputs.data_access = data_access`

File: pybossa/forms/dynamic_forms.py [hunk 2]
```diff
@@ -17,3 +20,27 @@ def build_field(field_type, description, validators, kwargs=None):
...
+
+
+    product = SelectField(lazy_gettext('Product'),
```
Comment (fiorda) [inline pybossa/forms/dynamic_forms.py]: Can these be added to the base class?

File: pybossa/view/projects.py [hunk 3]
```diff
@@ -309,6 +310,16 @@ def project_index(page, lookup, category, fallback, use_count, order_by=None,
         template_args.update({"count": count})
     return handle_content_type(template_args)
 
+def set_product_subproduct_choices(form):
```
Comment (fiorda) [inline pybossa/view/projects.py]: Could this be done in the `__init__` if you pass in the lists of products?

Comment (mariarivera30) [reply pybossa/view/projects.py]: So keep it as it was before but passing the products?

File: pybossa/forms/dynamic_forms.py (hunk 2 above)
Comment (mariarivera30) [reply pybossa/forms/dynamic_forms.py]: yes I thinks is there I forgot to remove it from here.

**Pull request merged**